tests/common/snappi_tests/common_helpers.py

```diff
@@ -107,13 +107,25 @@ def get_lossless_buffer_size(host_ans):
     """
     config_facts = host_ans.config_facts(host=host_ans.hostname,
                                          source="running")['ansible_facts']
+    # Fetching device_metadata for HWSKU info.
+    device_mtd = config_facts['DEVICE_METADATA']['localhost']['hwsku']
+
     is_cisco8000_platform = True if 'cisco-8000' in host_ans.facts['platform_asic'] else False
 
+    # Checking if platform is Nokia-7250
+    is_nokia7250_platform = True if ('Nokia' or 'nokia') and '7250' in device_mtd else False
```

Comment:
These changes to use the `egress_lossless_pool` instead of the ingress one should apply to all Broadcom DNX based platforms. Instead of calling out individual hardware SKUs, it would be better to use duthost.facts; e.g. 
```
duthost.facts["platform_asic"] == "broadcom-dnx"
```

Reply:
@amitpawar12 can you update the code to make it applicable for all broadcom platforms?